db/migrate/20220324071519_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true, index: true
+      t.references :post, null: false, foreign_key: true, index: true
+      t.string :Text
```

Comment:
- [ ] Please use a `text` instead of a `string` for the comment  because it may be longer

Reply:
alright @hiromataba  thanks for feedback